ModelBundle/Document/Node.php

```diff
@@ -374,6 +381,26 @@ public function getTheme()
     }
 
     /**
+     * Set default theme
+     *
+     * @param boolean $defaultTheme
+     */
+    public function setDefaultTheme($defaultTheme)
+    {
+        $this->defaultTheme = $defaultTheme;
```

Comment:
As defaultTheme must be boolean, you should check that it's the case before setting it.
Or cast it but it's ugly